src/wp-includes/cron.php

```diff
@@ -845,7 +845,7 @@ function spawn_cron( $gmt_time = 0 ) {
 	 * Multiple processes on multiple web servers can run this code concurrently,
 	 * this lock attempts to make spawning as atomic as possible.
 	 */
-	$lock = get_transient( 'doing_cron' );
+	$lock = (int) get_transient( 'doing_cron' );
```

Comment:
```suggestion
	$lock = (float) get_transient( 'doing_cron' );
```

The transient is set using `sprintf( '%.22F', $gmt_time );` so this is a float rather than an integer.